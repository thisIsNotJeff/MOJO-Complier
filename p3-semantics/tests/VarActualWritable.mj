const c = 0;
def foo(var i: int) {
  foo(c);
}