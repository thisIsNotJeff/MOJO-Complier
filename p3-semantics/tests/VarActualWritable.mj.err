 line 3, column 7: error: VAR actual must be a designator
