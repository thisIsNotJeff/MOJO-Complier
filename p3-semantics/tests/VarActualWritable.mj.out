BEGIN _BUILTIN
_BUILTIN.int=int
_BUILTIN.bool=bool
_BUILTIN.false:bool=false
_BUILTIN.true:bool=true
_BUILTIN.Null=^null
_BUILTIN.nil:Null=0
_BUILTIN.Refany=^null
_BUILTIN.Address=^null
_BUILTIN.Root=null
object {
}
_BUILTIN.Text=^null
_BUILTIN.First:(...)
_BUILTIN.Last:(...)
_BUILTIN.New:(...)
_BUILTIN.Number:(...)
BEGIN VarActualWritable
VarActualWritable.c:int=0
VarActualWritable.foo:(VAR i:int)
BEGIN VarActualWritable.foo
VarActualWritable.foo.i:int
END VarActualWritable.foo
END VarActualWritable
END _BUILTIN
